frontend/src/metabase/common/components/EntityPicker/components/EntityPickerModal/EntityPickerModal.tsx

```diff
@@ -47,6 +47,7 @@ export interface EntityPickerModalProps<Model extends string, Item> {
   searchResultFilter?: (results: Item[]) => Item[];
   actionButtons?: JSX.Element[];
   trapFocus?: boolean;
+  variant?: "large-overlay-padding";
```

Comment:
Maybe it should be in all cases for `EntityPickerModal`?

Reply:
Asked in [Slack](https://metaboat.slack.com/archives/C0645JP1W81/p1712674703445559?thread_ts=1712586211.035699&cid=C0645JP1W81).